packages/babel-parser/src/plugins/typescript.js

```diff
@@ -220,6 +220,26 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return result;
     }
 
+    tsParseImportType(allowReservedWords: boolean): N.TsImportType {
+      const node: N.TsImportType = this.startNode();
+      this.expect(tt._import);
+      this.expect(tt.parenL);
+      if (!this.match(tt.string)) {
+        throw this.unexpected();
+      }
+      node.argument = this.parseLiteral(this.state.value, "StringLiteral");
+      this.expect(tt.parenR);
+
+      if (this.match(tt.dot)) {
+        this.eat(tt.dot);
```

Comment:
You can directly use `if (this.eat(tt.dot))`

Reply:
i'm kinda new to this parser, thanks for suggestion